android/build.gradle

```diff
@@ -56,6 +56,6 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'  // From node_modules
-    implementation 'net.openid:appauth:0.11.1'
+    implementation 'net.openid:appauth:0.9.1'
```

Comment:
Any particular reason for downgrading the appauth library? It would be preferable to keep it at `0.11.1`

Reply:
@kadikraman Hello ! the downgrade is there because it's causing a conflict with the latest version of @react-native-firebase/perf which was giving me an " R8 RNAppAuthModule$5 is defined multiple times " error while running minify  using Hermes on release.

The error might be too specific , so i guess the version downgrade can be disregarded.

Reply:
Hm yes I'm not sure what would cause that. You can always use [patch-package](https://github.com/ds300/patch-package) to change the version for your project till a solution can be found.